docs/concepts/memap.md

```diff
@@ -64,20 +66,34 @@ The table above showed multiple memory sections.
 - ``.text``: is where the code application and constants are located in Flash.
 - ``.data``: nonzero initialized variables; allocated in both RAM and Flash memory (variables are copied from Flash to RAM at runtime).
 - ``.bss``: uninitialized data allocated in RAM, or variables initialized to zero.
-- ``Heap``: dynamic allocations in the Heap area in RAM (for example, used by ``malloc``). The maximum size value may be defined at build time.
-- ``Stack``: dynamic allocations in the Stack area in RAM (for example, used to store local data, temporary data when branching to a subroutine or context switch information). The maximum size value may be defined at build time.
+
+**Note**: This tool displays static RAM and Flash usage. The heap and stack space are not included in the report as are usually configured at run time, and it's likely to change from one application to another.
 
 There are other entries that require a bit of clarification:
 
 - Fill: represents the bytes in multiple sections (RAM and Flash) that the toolchain has filled with zeros because it requires subsequent data or code to be aligned appropriately in memory.
-- Misc: usually represents helper libraries introduced by the toolchain (like ``libc``), but can also represent modules that are not part of mbed.
+- Misc: usually represents helper libraries introduced by the toolchain but can also represent modules that are not part of mbed.
+
+## Other useful information
+
+It's possible to specify the depth value of the directory level when generating a report. By default *memap* is configured as `--depth=2`, but note that specifying `--depth=0` will display a full report. For example:
```

Comment:
- Please add a comma after "default" because it's an introductory prepositional phrase.
- Please change "will display" to "displays" for consistent tense with other documentation.